public/sassdoc.json

```diff
@@ -0,0 +1 @@
ustom value.\n","context":{"type":"mixin","name":"mdc-button-outline-width","code":"\n  // Note: Adjust padding to maintain consistent width with non-outlined buttons\n  $padding-value: max($padding - $outline-width, 0);\n\n  @include mdc-button-horizontal-padding($padding-value);\n\n  border-width: $outline-width;\n","line":{"start":99,"end":106}}}]},{"description":"Sets the outline width to the given number (defaults to 2px) and adjusts padding accordingly. `$padding` is only required in cases where `mdc-button-horizontal-padding` is also included with a custom value.\n","commentRange":{"start":98,"end":98},"context":{"type":"mixin","name":"mdc-button-outline-width","code":"\n  // Note: Adjust padding to maintain consistent width with non-outlined buttons\n  $padding-value: max($padding - $outline-width, 0);\n\n  @include mdc-button-horizontal-padding($padding-value);\n\n  border-width: $outline-width;\n","line":{"start":99,"end":106}},"group":["button"],"access":"public","require":[{"type":"mixin","name":"mdc-button-horizontal-padding"}],"file":{"path":"mdc-button/_mixins.scss","name":"_mixins.scss"}},{"description":"Sets the fill color of a card @group card @param {color} $color\n","commentRange":{"start":34,"end":34},"context":{"type":"mixin","name":"mdc-card-fill-color","code":"\n  @include mdc-theme-prop(background-color, $color);\n","line":{"start":36,"end":38}},"group":["card"],"access":"public","require":[],"file":{"path":"mdc-card/_mixins.scss","name":"_mixins.scss"}},{"description":"Sets the color and thickness of a card's outline (but does _not_ remove its shadow)\n","commentRange":{"start":40,"end":42},"context":{"type":"mixin","name":"mdc-card-outline","code":"\n  border: $thickness solid mdc-theme-prop-value($color);\n","line":{"start":43,"end":45}},"parameter":[{"type":"color","name":"color"},{"type":"pixel","name":"thickness"}],"group":["card"],"access":"public","require":[],"file":{"path":"mdc-card/_mixins.scss","name":"_mixins.scss"}},{"description":"Sets the corner radius of a card\n","commentRange":{"start":47,"end":48},"context":{"type":"mixin","name":"mdc-card-corner-radius","code":"\n  border-radius: $radius;\n","line":{"start":49,"end":51}},"parameter":[{"type":"pixel","name":"radius"}],"group":["card"],"access":"public","require":[],"file":{"path":"mdc-card/_mixins.scss","name":"_mixins.scss"}},{"description":"Maintains the given aspect ratio on a `mdc-card__media` subelement by dynamically scaling its height relative to its width\n","commentRange":{"start":53,"end":55},"context":{"type":"mixin","name":"mdc-card-media-aspect-ratio","code":"\n  &::before {\n    // This clever trick brought to you by: http://www.mademyday.de/css-height-equals-width-with-pure-css.html\n    margin-top: percentage($y / $x);\n  }\n","line":{"start":56,"end":61}},"parameter":[{"type":"number","name":"x"},{"type":"number","name":"y"}],"group":["card"],"access":"public","require":[],"file":{"path":"mdc-card/_mixins.scss","name":"_mixins.scss"}}]
```

Comment:
Is this always generated by `npm start` and `npm run build`? Should it be .gitignored?